exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/VendorDialect.kt

```diff
@@ -115,7 +115,7 @@ abstract class VendorDialect(
     }
 
     override fun existingIndices(vararg tables: Table): Map<Table, List<Index>> =
-        TransactionManager.current().db.metadata { existingIndices(*tables) }
+        TransactionManager.current().db.metadata { existingIndices(tables = tables.filter { it.exists() }.toTypedArray()) }
```

Comment:
Since the problem is with `MigrationUtils` and not with `SchemaUtils` (based on the test below passing if switched), would it be possible to address this issue closer to the source instead of at the higher `VendorDialect` level?

In the `SchemaUtils` function, `modifyTablesStatements` only uses tables that have been confirmed to exist, so I think it makes sense for the same be done for `MigrationUtils`? All the tests seem to pass with this line change:
```kt
val modifyTablesStatements = logTimeSpent("Checking mapping consistence", withLogs) {
    mappingConsistenceRequiredStatements(
        tables = tablesToAlter.toTypedArray(), // here
        withLogs
    ).filter { it !in (createStatements + alterStatements) }
}
```